### DISCUSSION TITLE
Command-line / API daemon version?

### DISCUSSION THREAD

**Discussion by jooray** [get10101/10101#1308] [Ideas]
I know you're mobile first, but Is there a command line interface? It could be useful for arbitrage and automatic hedging, maybe later running on the nodes such as umbrel. This could be later integrated into btcpayserver (for example to stabilize parts of income from e-shop) through btc transmuter.

But also market making, if there's a funding rate / contango-backwardation arbitration opportunity, it would be great to get liquidity from other exchanges through market-making / arbiteurs.

**Comment by Restioson** [comment]
Although we don't support it as a target, you can run the app on desktop at least locally for development purposes (not sure if it works on main net). @bonomat might be able to speak more about the API/automation side of things.

**Comment by bonomat** [comment]
This is definitely on our roadmap. You can follow the development of the [`maker` binary](https://github.com/get10101/10101/tree/main/maker) in this project which will evolve exactly into what you are asking for :)
